src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -119,6 +125,23 @@ internal static Features PowerFxV1AllowSetExpandedTypes
             CoalesceShortCircuit = true,
             PrimaryOutputPropertyCoercionDeprecated = true,
             AsTypeLegacyCheck = false,
+            JsonFunctionAcceptsLazyTypes = false
```

Comment:
>JsonFunctionAcceptsLazyTypes [](http://example.com/codeflow?start=12&length=28)

We can set it to true in PowerFxV1 - this won't break any existing scenarios (will only make current scenarios that don't work today work, so not a breaking change).